### ISSUE TITLE
cck 'recreate vm' requires 'serial' property

### ISSUE THREAD

**Issue by mariash** [cloudfoundry/bosh#1187]
Our deployment manifest does not specify 'serial' property (which should be optional based on https://bosh.io/docs/deployment-manifest.html). When running cck and choosing to recreate VM we are getting the following error:

```
E, [2016-03-25 23:49:10 #4047] [task:20] ERROR -- DirectorJobRunner: Error resolving problem `1': Property 'serial' (value nil) did not match the required type 'boolean'
E, [2016-03-25 23:49:10 #4047] [task:20] ERROR -- DirectorJobRunner: /var/vcap/packages/director/gem_home/ruby/2.1.0/gems/bosh-director-1.3215.0/lib/bosh/director/validation_helper.rb:50:in `invalid_type'
/var/vcap/packages/director/gem_home/ruby/2.1.0/gems/bosh-director-1.3215.0/lib/bosh/director/validation_helper.rb:16:in `safe_property'
/var/vcap/packages/director/gem_home/ruby/2.1.0/gems/bosh-director-1.3215.0/lib/bosh/director/deployment_plan/update_config.rb:46:in `initialize'
/var/vcap/packages/director/gem_home/ruby/2.1.0/gems/bosh-director-1.3215.0/lib/bosh/director/cloudcheck_helper.rb:86:in `new'
/var/vcap/packages/director/gem_home/ruby/2.1.0/gems/bosh-director-1.3215.0/lib/bosh/director/cloudcheck_helper.rb:86:in `block in recreate_vm'
/var/vcap/packages/director/gem_home/ruby/2.1.0/gems/bosh-director-1.3215.0/lib/bosh/director/instance_updater/instance_state.rb:5:in `with_instance_update'
/var/vcap/packages/director/gem_home/ruby/2.1.0/gems/bosh-director-1.3215.0/lib/bosh/director/cloudcheck_helper.rb:82:in `recreate_vm'
/var/vcap/packages/director/gem_home/ruby/2.1.0/gems/bosh-director-1.3215.0/lib/bosh/director/problem_handlers/unresponsive_agent.rb:37:in `block (2 levels) in <class:UnresponsiveAgent>'
```

**Comment by cppforlife** [comment]
Story: https://www.pivotaltracker.com/story/show/116469377

**Comment by cppforlife** [comment]
Fixed in 255.8
